build.gradle

```diff
@@ -757,17 +757,15 @@ task pythonIsInstalled(type: Exec) {
   args '--version'
 }
 
-task tags {
-  group 'Emacs'
+task tags(type: Exec, group: 'Emacs') {
```

Comment:
This should depend on `docTags` rather than the reverse.  No one will ever run `docTags` explicitly.